modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
```

Comment:
As written, this assumes the user has complete understanding of OIDC, as there's no mention of access tokens, ID Tokens, or any other critical piece of the protocol.